File: frontend/src/app/models/OperatingHours.ts
export class OperatingHours {
  id: string;
  restaurantId: string;
  dayOfWeek: string;
  opensAt: string;
  closesAt: string;

  constructor(
    id: string,
    restaurantId: string,
    dayOfWeek: string,
    opensAt: string,
    closesAt: string
  ) {
    this.id = id;
    this.restaurantId = restaurantId;
    this.dayOfWeek = dayOfWeek;
    this.opensAt = opensAt;
    this.closesAt = closesAt;
  }
}

File: backend/src/main/java/com/example/backend/models/OperatingHours.java
package com.example.backend.models;

public class OperatingHours {
    String id;
    String restaurantId;
    String dayOfWeek;
    String opensAt;
    String closesAt;

    public OperatingHours() {
    }

    public OperatingHours(String id, String restaurantId, String dayOfWeek, String opensAt, String closesAt) {
        this.id = id;
        this.restaurantId = restaurantId;
        this.dayOfWeek = dayOfWeek;
        this.opensAt = opensAt;
        this.closesAt = closesAt;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getRestaurantId() {
        return restaurantId;
    }

    public void setRestaurantId(String restaurantId) {
        this.restaurantId = restaurantId;
    }

    public String getDayOfWeek() {
        return dayOfWeek;
    }

    public void setDayOfWeek(String dayOfWeek) {
        this.dayOfWeek = dayOfWeek;
    }

    public String getOpensAt() {
        return opensAt;
    }

    public void setOpensAt(String opensAt) {
        this.opensAt = opensAt;
    }

    public String getClosesAt() {
        return closesAt;
    }

    public void setClosesAt(String closesAt) {
        this.closesAt = closesAt;
    }

}
